client/my-sites/sharing/connections/services/google-my-business.js

```diff
@@ -55,14 +68,9 @@ export class GoogleMyBusiness extends SharingService {
 	// override `removeConnection` to remove the keyring connection instead of the publicize one
 	removeConnection = () => {
 		this.setState( { isDisconnecting: true } );
-		this.props
-			.disconnectGoogleMyBusinessLocation(
-				this.props.siteId,
-				this.props.siteKeyrings[ 0 ].keyring_id
-			)
-			.finally( () => {
-				this.setState( { isDisconnecting: false } );
-			} );
+		this.props.disconnectAllGoogleMyBusinessAccounts( this.props.siteId ).finally( () => {
```

Comment:
`finally` is super new, are we transpiling that?
Otherwise, edge & safari [doesn't support it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally).

Reply:
I think we are, will need to check 👍